platform/source/tools/clustercli/common.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015-2017 Applatix, Inc. All rights reserved.
+#
+
+import re
+
+from anytree import Node
+from .cli import AbstractPrompt
+
+
+class CommonPrompts(AbstractPrompt):
+
+    INSTALLCMD = "install"
+    UNINSTALLCMD = "uninstall"
+
+    def __init__(self):
+        self.aws_profiles = self._get_profiles()
+        self._profile_names = [x for x in self.aws_profiles]
+
+        self.root = Node("name",
+                    prompt=u'Enter a cluster name',
+                    validator=u'^[a-zA-Z0-9-_]{2,20}$',
+                    help=u'This is a friendly name for your cluster. Minimum 2 characters, Maximum 20. Only a-z, A-Z, 0-9, - and _ allowed',
+                    default=u'argo-cluster'
+                    )
+
+        profileNode = Node("profile",
+                           prompt=u'Enter the AWS profile to use',
+                           values=self._profile_names,
```

Comment:
For generic defaults, please use class `ax.cluster_management.options.common.ClusterOperationDefaults`